include/librealsense2/h/rs_option.h

```diff
@@ -101,6 +101,7 @@ extern "C" {
         RS2_OPTION_SEQUENCE_NAME, /**< HDR Sequence size */
         RS2_OPTION_SEQUENCE_SIZE, /**< HDR Sequence size */
         RS2_OPTION_SEQUENCE_ID, /**< HDR Sequence ID - 0 is not HDR; sequence ID for HDR configuartion starts from 1 */
+        RS2_OPTION_HUM_TEMPERATURE, /**< Humidity temperature*/
```

Comment:
Please switch to full name HUMIDITY_TEMP(ERATURE)

Reply:
Done